bin/handlebarspp

```diff
@@ -13,29 +13,120 @@ This utility parses the handlebars template file and adds the context-sensitive
 */
 (function() {
 
-    var fs = require('fs'),
-        ContextParserHandlebars = require("../src/context-parser-handlebars.js"),
-        noOfArgs = process.argv.length, 
-        file;
-        
-    if (noOfArgs >= 3 && noOfArgs <= 4) {
-        file = process.argv[2];
-        if (fs.existsSync(file)) {
-            var parser = new ContextParserHandlebars({
-                printCharEnable: true,
-                strictMode: process.argv[3] === 'true',
-                processingFile: file
+    function printUsage() {
+        console.log("Usage: handlebarspp template...\n\n\
+Options:\n\
+\t-D\tDisable Partial Handling, and assume that partials are placed and will result in the text (aka HTML Data) context (default: false).\n\n\
+\t-p\tPartial Directory. The Secure Handlebars preloads the partials into the memory for partial parsing.\n\n\
+\t-e\tPartial File Extension. The partial file extension. (default: .handlebars)\n\n\
+\t-c\tMelt Partials into Template (experimental). The Secure Handlebars combines the partial contents into the input template. (default:false)\n\n\
+\t-d\tDump the preprocessed partials (prefixed with SJST/) into the partial directory\n\n\
```

Comment:
if u dump the combined template into this default folder SJST, please make it configurable.